app/package.json

```diff
@@ -59,7 +59,7 @@
       "/tests/"
     ],
     "moduleNameMapper": {
-      "\\.(css|scss|pcss|po|md|mdx)$": "identity-obj-proxy",
+      "\\.(css|scss|pcss|po|md)$": "identity-obj-proxy",
```

Comment:
Removed `mdx` extension, it wasn't used anywhere else was interfering with some mocks.